deep_quoridor/src/agents/alphazero.py

```diff
@@ -30,6 +37,119 @@ class AlphaZeroParams(SubargsBase):
     # Exploration vs exploitation.  0 is pure exploitation, infinite is random exploration.
     temperature: float = 1.0
 
+    # Number of searches
+    n: int = 1000
+    # A higher number favors exploration over exploitation
+    c: float = 1.4
+
+
+class AzNode:
```

Comment:
:donkey:

Reply:
Didn't notice, I'm loosing my wittiness.  BTW, at Facebook I once called a class HubsExactIndex, and a while later someone noticed it could be HubSexActIndex 🤣 .  I think linters should start using AI tools to check for those cases and give you a warning